RecFCCeeCalorimeter/src/components/CreateCaloCellPositionsFCCee.h

```diff
@@ -91,10 +92,10 @@ class CreateCaloCellPositionsFCCee : public GaudiAlgorithm {
   dd4hep::DDSegmentation::BitFieldCoder* m_decoder = new dd4hep::DDSegmentation::BitFieldCoder("system:4");
   /// Input collection
   DataHandle<edm4hep::CalorimeterHitCollection> m_hits{"hits/hits", Gaudi::DataHandle::Reader, this};
+  MetaDataHandle<std::string> m_hitsCellIDEncoding{m_hits, "CellIDEncodingString", Gaudi::DataHandle::Reader, this};
```

Comment:
Should MetaDataHandle take `fatherAlg`?

Reply:
Ah no. It seems like I didn't update this PR after that has been removed again from key4hep/k4FWCore#100. Let me fix that